taxi_service/settings.py

```diff
@@ -24,7 +24,7 @@
     "django-insecure-8ovil3xu6=eaoqd#-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc"
```

Comment:
The `SECRET_KEY` is hardcoded and exposed in the settings file. Consider using environment variables or a separate configuration file to keep it secure.